roxie/udplib/udpsha.cpp

```diff
@@ -51,7 +51,7 @@ unsigned udpLocalWriteSocketSize = 1024000;
 unsigned udpStatsReportInterval = 60000;
 
 unsigned udpOutQsPriority = 0;
-unsigned udpSendTraceThresholdMs = 5;
+unsigned udpSendTraceThresholdMs = 50;
```

Comment:
Increasing this because slightly paranoid that it will generate too much tracing when sending large amounts of data.